features/unattached_commands.feature

```diff
@@ -218,3 +218,69 @@ Feature: Command behaviour when unattached
           | impish  | disable  |
           | jammy   | enable   |
           | jammy   | disable  |
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Check for newer versions of the client in an ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        Then I verify that no files exist matching `/tmp/ubuntu-advantage/candidate-version`
+        When I run `pro status` as non-root
+        Then I will see the following on stderr
+        """
+        """
+        And I verify that files exist matching `/tmp/ubuntu-advantage/candidate-version`
+        # We forge a candidate to see results
+        When I delete the file `/tmp/ubuntu-advantage/candidate-version`
+        And I create the file `/tmp/ubuntu-advantage/candidate-version` with the following
+        """
+        99.9.9
+        """
+        And I run `chmod 777 /tmp/ubuntu-advantage/candidate-version` with sudo
+        And I run `pro status` as non-root
+        Then I will see the following on stderr
+        """
+        *Warning*: There is a new version of the client available:
+        99.9.9. Make sure to run
+        sudo apt install ubuntu-advantage-tools
+        to upgrade to the latest version and get all new features and bug fixes.
```

Comment:
Message bikeshedding thread:

I'd like to see the initial warning and version all on one line.
I'm not sure the `*` are necessary, but could be convinced either way
"Make sure to" can be replaced by "Please" IMO
Minor change ideas at the end, but they're very subjective and I don't have a strong opinion there
```suggestion
        Warning: A new version is available: 99.9.9.
        Please run:
            sudo apt install ubuntu-advantage-tools
        to get the latest version with new features and bug fixes.
```

Take or leave as much/little as you'd like from this suggestion

Reply:
I'll take everything! Except for the warning `*`s: @lucasmoura suggested those on the spec to "highlight" the word and catch the user's attention, and I think it makes sense.

Reply:
updated - let me know if I should change anything else

Reply:
We could also highlight the warning as we do with the commands on `fix`. My idea was just to have a visual cue for the user to not ignore that message. But I don't have a strong opinion here

Reply:
I also don't have a strong opinion. We could highlight it by making it a color.

Now that I was playing with pip for the other comment, theirs is a "notice", not a warning, and they have "notice" in blue.
```
[notice] A new release of pip available: 22.1.2 -> 22.2.2
[notice] To update, run: pip install --upgrade pip
```

npm also uses the word "notice" and they color it teal (in my terminal at least, not sure what terminal color it is - maybe "bright green"?)
```
changed 1 package, and audited 2 packages in 2s

found 0 vulnerabilities
npm notice 
npm notice New minor version of npm available! 8.5.1 -> 8.17.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.17.0
npm notice Run npm install -g npm@8.17.0 to update!
npm notice 
```

So maybe we shouldn't even call it a warning? We could say "Info" in blue (since we already have a specific meaning for "Notice" in status output).

Sorry I'm adding fuel to the flaming bike shed. :fire: :bike: :house: :upside_down_face:

Reply:
Blue [info] is there!